org.opent2t.sample.thermostat.superpopular/com.wink.thermostat/js/README.md

```diff
@@ -55,7 +55,7 @@ Please enter credentials for the Wink API:
 
 Thanks! Signing you in to Wink.
 Signed in to WINK.
-? Which device do you want to onboard? Jaffri Home Entryway Thermostat (137418)
+? Which device do you want to onboard? Home Entryway Thermostat (137418)
   access_token : abcc2f4ds55asd531ec78cc08_l4Qwj

```

Comment:
Yes this is not a valid token... I'll change it to a placeholder in an update to this PR.